table/block_based_table_factory.cc

```diff
@@ -139,6 +139,11 @@ std::string BlockBasedTableFactory::GetPrintableTableOptions() const {
              table_options_.block_cache->GetCapacity());
     ret.append(buffer);
   }
+  if (table_options_.persistent_cache) {
```

Comment:
actually we do print if `table_options_.block_cache` is `nullptr` in line 134
can we do the same here ?